### ISSUE TITLE
Installation instructions

### ISSUE THREAD

**Issue by CaCu15** [Kane610/deconz#3]
Greetings Kane,

just found this project and it looks exactly like the thing I was searching for. I already own an RaspBee gateway running DECONZ and some IKEA Tradfri equipment. This seems to be the solution to integrate a broad range of ZigBee devices into HA. 
Hence I would like to try your solution, but I can't figure out, how to install it on HASS.IO on RPI3.  Can I simply copy the files to  config/custom_components using the configurator component? What else is required for setup?

Thanks in advance for your support
Carsten

**Comment by Kane610** [comment]
Glad to hear you might find this project usable.
Since hass.io is using docker there is no easy way for you to add this component yourself until it has been merged and released in HASS. 

For deconz software and hardware on the same device as hass.io there are some additional issues with the communication between software and hardware that is being worked on by another community member.

Join us on the hass forum for a better dialogue. https://community.home-assistant.io/t/conbee-zigbee-stick-and-ha-will-this-work-together/15535/243

**Comment by CaCu15** [comment]
Thanks for your response. 
OK, than I will wait for this to be released (although it's hard, because I would have liked to spent my Christmas holidays with this.) . Do you have any idea, how long it will take? Is it going to take some weeks or are we talking about months?  
I will join the dialogue linked above for further information.

Greetings, Carsten

**Comment by CaCu15** [comment]
Hi Kane,

unfortunately the forum linked above is in read-only mode, hence I can not contribute there. Hence I'm writing here.
I managed to get pydeconz working on my HASS.IO setup using the docker image from here: 
[https://hub.docker.com/r/marthoc/home-assistant-deconz/](https://hub.docker.com/r/marthoc/home-assistant-deconz/).  It's running on a Debian 9 VM, DECONZ is running on a P3.
I can see my Tradfri motion sensor and remote on the main page of Home Assistant and I can switch the IKEA bulbs linked to DECONZ. But I'm facing a problem: As soon as the docker container for pydeconz is running, the HASSIO menu item i gone. If I disable this docker container (it's started as a debian service) the menu item appears again. Somehow the docker container seems to occupy IP ports that the HASSIO add-on want's to use as well. Additionally there are some error messages in the home-assistant.log (see attachment) saying that the component DECONZ could not be found.

[homeassistant.log](https://github.com/Kane610/deconz/files/1584133/homeassistant.log)

**Comment by Kane610** [comment]
Hmm, do you have an account with Hass forums? That should be the only requirement to join the discussion.

Unfortunately I don't have much knowledge about docker and it is @marthoc who has been putting in the hours with the docker so I must refer you to him with questions specific to hass.io and containers.

**Comment by marthoc** [comment]
The image from this repo: https://hub.docker.com/r/marthoc/home-assistant-deconz/ is a full Home Assistant installation containing the deconz component on top of it. So yes, if you are running that image, you are not using Hass.io and will not have a Hass.io menu item. 

It may be possible to replace the Hass.io Home Assistant image with my rpi variant image, but I can’t put any time into this for the next weeks and it is likely the case that @Kane610’s component is going to be merged soon and in that case it will be in thr next release of Hassio in any event. 

Hope this helps!

**Comment by marthoc** [comment]
To be clear - my images are not Hass.io addons. They are full images of Home Assistant. When that image is running, it is occupying port 8123 and seemingly overriding your Hassio.

**Comment by CaCu15** [comment]
Hello Mark, Hello Kane,

thx for your support. Finally I figured out that your image, Mark, was in conflict with the HASSIO container. I kicked HASSIO out for the moment and set everything up on plain docker. Now I'm using your docker image, Mark, together with images for MQTT and Node-Red (notoriousbdg/hassio-addon-node-red-amd64:latest,  toke/mosquitto) and everything seems to work fine. 
I'm currently tring to build some Node-Red flows using your DECONZ components, especially I would like to figure out how the TradFri remote can be used this way.  

Btw: I still can't sign-up for the HASSIO forum linked above. If I hit he "sign up" or "Login" Buttons in the top of the screen, I get the message "Login is disabled while the site is in read only mode".
